packages/treasure/src/test-transactions.ts

```diff
@@ -24,6 +31,24 @@ export const MINT_TREASURE_TAG: TestParams<MintActionParams> = {
   },
 }
 
+export const STAKE_MAGIC: TestParams<StakeActionParams> = {
+  transaction: {
+    chainId: Chains.ARBITRUM_ONE,
+    from: '0x4ec63eefb25067d9ac0c54ca1d10f00930c599d9',
+    hash: '0xeb43c2bdfa3ac3a13061dde88b098f7acf121848cd8fb5a5f0c7e18b1fa76c48',
+    input:
+      '0xb6b55f250000000000000000000000000000000000000000000000001bc16d674ec80000',
+    to: MAGIC_STAKING,
+    value: '0',
+  },
+  params: {
+    chainId: Chains.ARBITRUM_ONE,
+    contractAddress: MAGIC_STAKING,
+    tokenOne: MAGIC,
+    amountOne: '2000000000000000000',
+  },
+}
+
 export const EXACT_TOKENS_FOR_TOKENS: TestParams<SwapActionParams> = {
   transaction: {
     chainId: 42161,
```

Comment:
[nit] I think we might as well update the rest of the chainIds to also use `Chains.ARBITRUM_ONE`

Reply:
Updated! I wasn't sure if there was a specific reason for the txns having hard-coded values